src/mega-plusplus.coffee

```diff
@@ -111,6 +110,12 @@ module.exports = (robot) ->
         from:      from
       }
 
+  robot.hear(new RegExp(plusPlusRegex, 'ig'), (msg) ->
```

Comment:
doesn't using `///` already make it a regex? Do we need both `///` and `new RegExp`? If we use `new`, can we `new` it just just once instead of every time we get a message?

also when we define plusPlusRegex, it already uses case insensitive. Can we add the `g` designation there or only define them here?

```
plusPlusRegex = ///
...
///i
```

Do we need

Reply:
Can probably just `new` it once, I would assume. 

I'll have to revisit the i/g designations to try and remember why I only used `ig` in the one place.